pages/linux.tsx

```diff
@@ -7,40 +8,50 @@ function Page() {
 
 export async function getServerSideProps({ res }: {res: NextApiResponse}) {
 
-  const fetched = await fetch('https://api.github.com/repos/oxen-io/lokinet/releases/latest');
-  if (!fetched) {
-    return {
-      notFound: true,
-    }
-  }
-  const json = await fetched.json()
-  if (!json || !json.assets) {
-    return {
-      notFound: true,
-    }
-  }
-  const assetsUrl =  json.assets.map((m:any) => m.browser_download_url)
-  if (!assetsUrl || !assetsUrl.length) {
-    return {
-      notFound: true,
-    }
+  var downloadURL = await GetDownloadURL("latest", ".tar.xz");
```

Comment:
we have linux packages repos at deb.oxen.io and rpm.oxen.io
i'd suggest making a new page with instructions on how to use both as the github releases are just signed source code tarballs.